reporter/datadog_reporter.go

```diff
@@ -323,7 +323,7 @@ func (r *DatadogReporter) reportProfile(ctx context.Context) error {
 	}
 
 	tags := strings.Split(config.ValidatedTags(), ";")
-	tags = append(tags, "runtime:go")
+	tags = append(tags, "runtime:native")
```

Comment:
We might want to change the actual runtime at one point if we want to make special rules to avoid having a service view.
Though I am fine with this :+1: